curation/uta_services.py

```diff
@@ -127,7 +127,10 @@ def get_genomic_coords(self, tx_ac, start_exon, end_exon,
         :return: Dictionary containing transcript and exon data
         """
         if gene:
-            gene = gene.upper()
+            gene = gene.upper().strip()
```

Comment:
Gonna leave this here for now, but in a future issue, I think we should move input format validation (strip, quote/dequote) to the views methods and have the service classes assume input is well-formed

Reply:
Okay! I didn't know if someone would be calling these methods directly or not. I did it here to cover both cases, but we can move it